RELEASE.md

```diff
@@ -1,78 +1,164 @@
-[add sha - version](https://github.com/kubernetes/k8s.io/blob/main/k8s.gcr.io/images/k8s-staging-ingress-nginx/images.yaml#L1)
+C. Wait for [cloud build](https://console.cloud.google.com/cloud-build/builds?project=k8s-staging-ingress-nginx)
+
+D. Promote Images:
+
+  Open pull request to promote staging image:
+  [add sha - version](https://github.com/kubernetes/k8s.io/blob/main/k8s.gcr.io/images/k8s-staging-ingress-nginx/images.yaml#L1)
 
 The sha is available in output from [cloud build](https://console.cloud.google.com/cloud-build/builds?project=k8s-staging-ingress-nginx)
 
-2. Change to images:
+## 2. Change to other Images:
+
    
-* [e2e](https://github.com/kubernetes/ingress-nginx/tree/main/images/test-runner)
+A. [e2e](https://github.com/kubernetes/ingress-nginx/tree/main/test/e2e-image)
+
+B. [test-runner](https://github.com/kubernetes/ingress-nginx/tree/main/images/test-runner)
+C. [echo](https://github.com/kubernetes/ingress-nginx/tree/main/images/echo)
+
+D. [cfssl](https://github.com/kubernetes/ingress-nginx/tree/main/images/cfssl)
+
+E. [fastcgi-helloserver](https://github.com/kubernetes/ingress-nginx/tree/main/images/fastcgi-helloserver)
 
-    * [test-runner](https://github.com/kubernetes/ingress-nginx/tree/main/images/echo)
-    * [echo](https://github.com/kubernetes/ingress-nginx/tree/main/images/echo)
-    * [cfssl](https://github.com/kubernetes/ingress-nginx/tree/main/images/cfssl)
-    * [fastcgi-helloserver](https://github.com/kubernetes/ingress-nginx/tree/main/images/fastcgi-helloserver)
-    * [httpbin](https://github.com/kubernetes/ingress-nginx/tree/main/images/httpbin)
+F. [httpbin](https://github.com/kubernetes/ingress-nginx/tree/main/images/httpbin)
 
-* Open pull request
-* Merge
-* Wait for [cloud build](https://console.cloud.google.com/cloud-build/builds?project=k8s-staging-ingress-nginx)
+G. Open pull request
 
-2a. Promote images:
+H. Merge
 
-* Open pull request to promote [staging image](https://github.com/kubernetes/k8s.io/blob/main/k8s.gcr.io/images/k8s-staging-ingress-nginx/images.yaml)
+I. Wait for [cloud build](https://console.cloud.google.com/cloud-build/builds?project=k8s-staging-ingress-nginx)
+
+J. Promote Images:
+
+- Open pull request to promote [staging image](https://github.com/kubernetes/k8s.io/blob/main/k8s.gcr.io/images/k8s-staging-ingress-nginx/images.yaml)
     * e2e-test-runner
     * e2e-test-cfssl
     * e2e-test-echo
     * e2e-test-fastcgi-helloserver
     * e2e-test-httpbin
 
-3. Update references to e2e-test-runner image:
+## 3. Update references to e2e-test-runner image:
 
 * [e2e-image](https://github.com/kubernetes/ingress-nginx/blob/main/test/e2e-image/Dockerfile#L1)
-* [run-in-docker.sh](https://github.com/kubernetes/ingress-nginx/blob/ff60aa9e2b5377db1544091b98f475a90a630297/build/run-in-docker.sh#L37)
+* [run-in-docker.sh](https://github.com/kubernetes/ingress-nginx/blob/main/build/run-in-docker.sh#L37)
+
+## 4. Promote Controller Image:
```

Comment:
If we make the changes to step 1, this step is redundant. We have already promoted the image for the controller.